changelog.md

```diff
@@ -0,0 +1,19 @@
+Harvest [bridge-adapter] (2019-09-06)
+  * \[kinetic-bridge-adapter-harvest]
+    * Refactored the bridge to leverage the harvest V2 endpoint.
+
+Harvest [bridge-adapter] (2019-12-31)
+  * \[kinetic-bridge-adapter-harvest]
+    * All unit tests passed or skipped.
+
+Harvest [bridge-adapter] (2020-06-09)
+  * \[kinetic-bridge-adapter-harvest] [v2.1.0]
+    * Refactored the adapter to leverage a different "adapter api".
+
+Harvest [bridge-adapter] (2020-06-24)
+  * \[kinetic-bridge-adapter-harvest] [v2.1.0]
+    * Removed the lib dir and updated after merge.
+
+Harvest [bridge-adapter] (2020-07-24)

```

Comment:
Combine all changes under one version, not date as they were not previously released.